### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New cal csvs for instruments returned from vendor refurb.
Also includes 6 new cal csvs from NUTNR post-cruise recovery testing (3 are older ones from Pioneer 15 recovery & 3 are from Pioneer 16 recovery).

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20210416.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
....92, 274.74, 275.55, 276.37, 277.18, 278, 278.81, 279.63, 280.44, 281.26, 282.07, 282.88, 283.7, 284.51, 285.33, 286.14, 286.96, 287.77, 288.59, 289.4, 290.22, 291.03, 291.85, 292.66, 293.48, 294.29, 295.11, 295.92, 296.74, 297.55, 298.37, 299.18, 299.99, 300.81, 301.62, 302.44, 303.25, 304.07, 304.88, 305.69, 306.51, 307.32, 308.14, 308.95, 309.76, 310.58, 311.39, 312.21, 313.02, 313.83, 314.65, 315.46, 316.27, 317.09, 317.9, 318.71, 319.53, 320.34, 321.15, 321.96, 322.78, 323.59, 324.4, 325.21, 326.03, 326.84, 327.65, 328.46, 329.27, 330.09, 330.9, 331.71, 332.52, 333.33, 334.14, 334.95, 335.77, 336.58, 337.39, 338.2, 339.01, 339.82, 340.63, 341.44, 342.25, 343.06, 343.87, 344.68, 345.49, 346.3, 347.11, 347.91, 348.72, 349.53, 350.34, 351.15, 351.96, 352.76, 353.57, 354.38, 355.19, 355.99, 356.8, 357.61, 358.42, 359.22, 360.03, 360.84, 361.64, 362.45, 363.25, 364.06, 364.86, 365.67, 366.47, 367.28, 368.08, 368.89, 369.69, 370.5, 371.3, 372.1, 372.91, 373.71, 374.51, 375.32, 376.12, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.73, 382.53, 383.33, 384.13, 384.93, 385.73, 386.53, 387.33, 388.13, 388.93, 389.73, 390.53, 391.32, 392.12, 392.92, 393.72, 394.51, 395.31, 396.11]",
```
Comment (batryn) [inline calibration/NUTNRB/CGINS-NUTNRB-01065__20210416.csv:8]: Spot checked. Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20210416.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...272.64, 273.44, 274.25, 275.05, 275.85, 276.66, 277.46, 278.26, 279.07, 279.87, 280.67, 281.48, 282.28, 283.08, 283.89, 284.69, 285.49, 286.3, 287.1, 287.91, 288.71, 289.51, 290.32, 291.12, 291.92, 292.73, 293.53, 294.33, 295.14, 295.94, 296.74, 297.55, 298.35, 299.15, 299.96, 300.76, 301.56, 302.37, 303.17, 303.97, 304.78, 305.58, 306.38, 307.19, 307.99, 308.79, 309.59, 310.4, 311.2, 312, 312.81, 313.61, 314.41, 315.21, 316.02, 316.82, 317.62, 318.42, 319.22, 320.03, 320.83, 321.63, 322.43, 323.23, 324.04, 324.84, 325.64, 326.44, 327.24, 328.04, 328.84, 329.64, 330.45, 331.25, 332.05, 332.85, 333.65, 334.45, 335.25, 336.05, 336.85, 337.65, 338.45, 339.25, 340.05, 340.85, 341.65, 342.45, 343.25, 344.04, 344.84, 345.64, 346.44, 347.24, 348.04, 348.84, 349.63, 350.43, 351.23, 352.03, 352.82, 353.62, 354.42, 355.22, 356.01, 356.81, 357.61, 358.4, 359.2, 360, 360.79, 361.59, 362.38, 363.18, 363.97, 364.77, 365.56, 366.36, 367.15, 367.95, 368.74, 369.53, 370.33, 371.12, 371.92, 372.71, 373.5, 374.29, 375.09, 375.88, 376.67, 377.46, 378.26, 379.05, 379.84, 380.63, 381.42, 382.21, 383, 383.79, 384.58, 385.37, 386.16, 386.95, 387.74, 388.53, 389.32, 390.11, 390.9, 391.68, 392.47, 393.26]",
```
Comment (batryn) [inline calibration/NUTNRB/CGINS-NUTNRB-01103__20210416.csv:8]: Spot checked. Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20201204.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...8, 273.28, 274.09, 274.89, 275.7, 276.5, 277.31, 278.11, 278.92, 279.72, 280.53, 281.33, 282.14, 282.94, 283.75, 284.56, 285.36, 286.17, 286.97, 287.78, 288.58, 289.39, 290.2, 291, 291.81, 292.61, 293.42, 294.22, 295.03, 295.84, 296.64, 297.45, 298.25, 299.06, 299.86, 300.67, 301.48, 302.28, 303.09, 303.89, 304.7, 305.5, 306.31, 307.12, 307.92, 308.73, 309.53, 310.34, 311.14, 311.95, 312.75, 313.56, 314.36, 315.17, 315.97, 316.78, 317.58, 318.39, 319.19, 320, 320.8, 321.61, 322.41, 323.22, 324.02, 324.83, 325.63, 326.44, 327.24, 328.04, 328.85, 329.65, 330.46, 331.26, 332.06, 332.87, 333.67, 334.48, 335.28, 336.08, 336.89, 337.69, 338.49, 339.3, 340.1, 340.9, 341.7, 342.51, 343.31, 344.11, 344.91, 345.72, 346.52, 347.32, 348.12, 348.92, 349.72, 350.53, 351.33, 352.13, 352.93, 353.73, 354.53, 355.33, 356.13, 356.93, 357.73, 358.53, 359.33, 360.13, 360.93, 361.73, 362.53, 363.33, 364.13, 364.93, 365.73, 366.53, 367.32, 368.12, 368.92, 369.72, 370.52, 371.31, 372.11, 372.91, 373.71, 374.5, 375.3, 376.1, 376.89, 377.69, 378.48, 379.28, 380.08, 380.87, 381.67, 382.46, 383.26, 384.05, 384.84, 385.64, 386.43, 387.23, 388.02, 388.81, 389.61, 390.4, 391.19, 391.98, 392.78, 393.57, 394.36]",
```
Comment (batryn) [inline calibration/NUTNRB/CGINS-NUTNRB-01106__20201204.csv:8]: Spot checked. Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20210416.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (batryn) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20210416.csv:8]: Spot checked. Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Comment by rtravis-whoi** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20210416.csv (hunk 2 above)
Comment (cdobs) [reply calibration/NUTNRB/CGINS-NUTNRB-01103__20210416.csv:8]: Spot check looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20201204.csv (hunk 3 above)
Comment (cdobs) [reply calibration/NUTNRB/CGINS-NUTNRB-01106__20201204.csv:8]: Spot check looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20210416.csv (hunk 4 above)
Comment (cdobs) [reply calibration/NUTNRB/CGINS-NUTNRB-01107__20210416.csv:8]: Spot check looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0096__20210326.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0096,CC_cala,0.0993,source_file PCO2W-B_SAMI-CO2_SN_C0096_Calibration_Files_2021-03-26.zip > c96_032121_CalCert.pdf
+C0096,CC_calb,0.3355,
+C0096,CC_calc,-1.1020,
+C0096,CC_calt,11.4443,
+C0096,CC_ea434,19706,Constant
+C0096,CC_ea620,34,Constant
+C0096,CC_eb434,3073,Constant
+C0096,CC_eb620,44327,Constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PCO2WB/CGINS-PCO2WB-C0096__20210326.csv:9]: Looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0105__20210326.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0105,CC_cala,0.0892,source_file PCO2W-C_SAMI-CO2_SN_C0105_Calibration_Certificate_2021-03-26.pdf
+C0105,CC_calb,0.4068,
+C0105,CC_calc,-1.2582,
+C0105,CC_calt,10.7716,
+C0105,CC_ea434,19706,constant
+C0105,CC_ea620,34,constant
+C0105,CC_eb434,3073,constant
+C0105,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PCO2WC/CGINS-PCO2WC-C0105__20210326.csv:9]: Looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0121__20210405.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0121,CC_cala,0.0771,source_file PCO2W-C_SAMI-CO2_SN_C0121_Calibration_Files_2021-04-05.zip > c121_033121E_CalCert.pdf
+C0121,CC_calb,0.4652,
+C0121,CC_calc,-1.2960,
+C0121,CC_calt,11.0278,
+C0121,CC_ea434,19706,constant
+C0121,CC_ea620,34,constant
+C0121,CC_eb434,3073,constant
+C0121,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PCO2WC/CGINS-PCO2WC-C0121__20210405.csv:9]: Looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0097__20210326.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0097,CC_cala,0.1483, source_file PCO2W-B_SAMI-CO2_SN_C0097_Calibration_Files_2021-03-26.zip > c97_032121_CalCert.pdf
+C0097,CC_calb,0.0932,
+C0097,CC_calc,-0.8345,
+C0097,CC_calt,10.7955,
+C0097,CC_ea434,19706,constant
+C0097,CC_ea620,34,constant
+C0097,CC_eb434,3073,constant
+C0097,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PCO2WB/CGINS-PCO2WB-C0097__20210326.csv:9]: Looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0122__20210312.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0122,CC_cala,0.0534,source_file PCO2W-C_SAMI-CO2_SN_C0122_Calibration_Files_2021-03-12.zip > SAMI_120321_CalCert.pdf
+C0122,CC_calb,0.6125,
+C0122,CC_calc,-1.5433,
+C0122,CC_calt,10.8462,
+C0122,CC_ea434,19706,constant
+C0122,CC_ea620,34,constant
+C0122,CC_eb434,3073,constant
+C0122,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WC/CGINS-PCO2WC-C0122__20210312.csv:9]: Looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0139__20210312.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0139,CC_cala,0.0394,source_file PCO2W-C_SAMI-CO2_SN_C0139_Calibration_Files_2021-03-12.zip > SAMI_120321_CalCert.pdf
+C0139,CC_calb,0.6568,
+C0139,CC_calc,-1.4995,
+C0139,CC_calt,10.7855,
+C0139,CC_ea434,19706,constant
+C0139,CC_ea620,34,constant
+C0139,CC_eb434,3073,constant
+C0139,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PCO2WC/CGINS-PCO2WC-C0139__20210312.csv:9]: Looks good

File: calibration/PHSENE/CGINS-PHSENE-P0140__20210326.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0140,CC_ea434,17372.0,source_file PHSEN-E_SAMI-pH_SN_P0140_Validation_Files_2021-03-26.zip > SAMI_P0140_260321_tris_ValCert.pdf
+P0140,CC_ea578,94.1,
+P0140,CC_eb434,2284.0,
+P0140,CC_eb578,38676.0,
+P0140,CC_ind_off,0,
+P0140,CC_ind_slp,1,
+P0140,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0140__20210326.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0081__20210309.csv [hunk 12]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0081,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0081_Validation_Files_2021-03-09.zip > SAMI_P0081_090321_ValCert.pdf
+P0081,CC_ea578,94.1,
+P0081,CC_eb434,2284.0,
+P0081,CC_eb578,38676.0,
+P0081,CC_ind_off,0,
+P0081,CC_ind_slp,1,
+P0081,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0081__20210309.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0142__20210316.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0142,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0142_Validation_Files_2021-03-16.zip > SAMI_P0142_160321t_ValCert.pdf
+P0142,CC_ea578,94.1,
+P0142,CC_eb434,2284.0,
+P0142,CC_eb578,38676.0,
+P0142,CC_ind_off,0,
+P0142,CC_ind_slp,1,
+P0142,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0142__20210316.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0144__20210326.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0144,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0144_Validation_Files_2021-03-26.zip > SAMI_P0144_260321_tris_ValCert.pdf
+P0144,CC_ea578,94.1,
+P0144,CC_eb434,2284.0,
+P0144,CC_eb578,38676.0,
+P0144,CC_ind_off,0,
+P0144,CC_ind_slp,1,
+P0144,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0144__20210326.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0180__20210316.csv [hunk 15]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0180,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0180_Validation_Files_2021-03-16.zip > SAMI_P0180_160321t_ValCert.pdf
+P0180,CC_ea578,94.1,
+P0180,CC_eb434,2284.0,
+P0180,CC_eb578,38676.0,
+P0180,CC_ind_off,0,
+P0180,CC_ind_slp,1,
+P0180,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0180__20210316.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0183__20210309.csv [hunk 16]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0183,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0183_Validation_Files_2021-03-09.zip > SAMI_P0183_090321_ValCert.pdf
+P0183,CC_ea578,94.1,
+P0183,CC_eb434,2284.0,
+P0183,CC_eb578,38676.0,
+P0183,CC_ind_off,0,
+P0183,CC_ind_slp,1,
+P0183,CC_psal,35,Constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PHSENF/CGINS-PHSENF-P0183__20210309.csv:8]: Looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00238__20210112.csv [hunk 17]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+238,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347] ",source_file SPKIR-B_OCR-507_SN_238_Calibration_Files_2021-01-12.zip > DI7238E.cal
+238,CC_offset,"[2147475429.7, 2147867339.7, 2147813410.5, 2147336096.0, 2146669273.2, 2147793594.3, 2147036115.2] ",
+238,CC_scale,"[2.14591849368e-007, 2.00707674801e-007, 2.04575797481e-007, 2.04259803491e-007, 2.1434970421e-007, 2.11545521529e-007, 2.07648137935e-007] ",
```
Comment (cdobs) [inline calibration/SPKIRB/CGINS-SPKIRB-00238__20210112.csv:4]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good to me

File: calibration/CTDBPC/CGINS-CTDBPC-50056__20210226.csv [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+16-50056,CC_pa2,-7.548720e-13,
+16-50056,CC_ptca0,5.093621e+05,
+16-50056,CC_ptca1,-4.008702e+01,
+16-50056,CC_ptca2,6.497084e-01,
+16-50056,CC_ptcb0,2.495013e+01,
+16-50056,CC_ptcb1,-9.750000e-04,
+16-50056,CC_ptcb2,0.000000e+00,
+16-50056,CC_ptempa0,-6.550438e+01,
+16-50056,CC_ptempa1,5.243691e+01,
+16-50056,CC_ptempa2,-5.228567e-01,
```
Comment (batryn) [inline calibration/CTDBPC/CGINS-CTDBPC-50056__20210226.csv:23]: Looks good

**Review by batryn — COMMENTED** [review]
Reviewed last file. Looks good

**Pull request merged**